### DISCUSSION TITLE
Incoming Updates: Organization, Community and Ease of Use

### DISCUSSION THREAD

**Discussion by debanjum** [khoj-ai/khoj#229] [Announcements]
We have a couple of interesting updates to share!

1. ☁️ For folks who don't want to self host. We're creating a cloud-hosted service for Khoj
    You can sign up for early access to it on our ✨ shiny new website at [https://khoj.dev](https://khoj.dev)!
2. 🔌 Two new plugins are going be added to the next release: **PDFs** and **Github**.
   1. PDFs: The PDF plugin will be able to index all the text (not images) from your PDF documents
   2. Github: This plugin will index all commit messages and markdown files in a Github repository
3. 🍬 We've updated the config page on the Khoj web interface. It was previously hideous and altogether unusable. It is now less hideous and quite usable.
4. 💬 We've created a [Discord channel](https://discord.com/invite/fP89zSJb) where you can provide feedback on what direction Khoj should take in the future. We would love to create a community of active users and contributors.
5. 🏡 The Khoj repository will be migrating into the khoj-ai organization: https://github.com/khoj-ai/
6. 👷🏽 We, @sabaimran and @debanjum, are going to be working together on Khoj full time for a bit to see if we can make Khoj useful for enough people.

Hope to see you all around on [Discord](https://discord.com/invite/BDgyabRM6e) soon 🚀

**Comment by Jawabiscuit** [comment]
I'm getting "invalid invite" when clicking on the Discord links in this message.

**Reply by debanjum** [reply]
Ah, sorry about that. Try see if this updated link works: https://discord.com/invite/BDgyabRM6e?

Edit: Thanks for notifying, I've updated the link in the announcement to the new invite link as well

**Comment by Jawabiscuit** [comment]
Worked!

Jonas

Sent from Proton Mail mobile

-------- Original Message --------
